data/vendors.json

```diff
@@ -33598,5 +33598,29 @@
       "ru": "AnSpacePay",
       "uk": "AnSpacePay"
     }
+  },
+  {
+    "code": "onlypay",
+    "status": "active",
+    "links": {
+      "website": "https://onlypay.ltd"
+    },
+    "name": {
+      "en": "Onlypay",
+      "ru": "Onlypay",
+      "uk": "Onlypay"
+    }
+  },
+  {
+    "code": "thepaymentconsultant",
+    "status": "active",
+    "links": {
+      "website": "https://thepaymentconsultant.com/"
+    },
+    "name": {
+      "en": "The payment consultant",
```

Comment:
The Payment Consultant
![image](https://github.com/openfintechio/openfintech/assets/28463622/f683e898-d11f-43b3-ae42-8cca388997ae)
![image](https://github.com/openfintechio/openfintech/assets/28463622/f13ae8b8-cad6-4890-a86f-9642b339a44a)